src/config/passport/passport.js

```diff
@@ -0,0 +1,41 @@
+import passport from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import { unhashedPassword } from '../../helpers/auth';
+import { User } from '../../sequelize/models';
+
+const passportConfig = () => {
```

Comment:
I don't see the jsDocs comments for this function like it's the case for all other functions. We do well to document our functions that way because it serves as a reminder (to you and to other developers looking at your code) of what the function actually does. Just like you did here for the login function
<img width="443" alt="Screen Shot 2019-08-11 at 00 19 19" src="https://user-images.githubusercontent.com/50101879/62827560-4b749000-bbd1-11e9-8694-680aba39e32c.png">

Reply:
well noted